pkgs/fc/agent/fc/util/enc.py

```diff
@@ -30,13 +30,43 @@ def load_enc(log, enc_path):
     return enc
 
 
-def seed_enc(tmpdir, enc_path):
-    tmp_enc_path = tmpdir / "fc-data/enc.json"
-    if not enc_path.exists():
-        return
-    if not tmp_enc_path.exists():
+def seed_enc(log, tmpdir, enc_path):
+    """Populates ENC data during bootstrapping. Our automation puts an
+    initial ENC file in /tmp which provides the password to allow the agent to
```

Comment:
phrasing: This initial file is the "seeding" referred to later in the logging messages, correct? The same terminology should be used in the docstring as well to link these together.

```suggestion
    initial ENC file in /tmp for seeding which provides the password to allow the agent to
```